cmake/linker_script/arm/linker.cmake

```diff
@@ -148,6 +148,7 @@ zephyr_linker_symbol(SYMBOL __kernel_ram_start EXPR "(@__bss_start@)")
 zephyr_linker_symbol(SYMBOL __kernel_ram_end  EXPR "(${RAM_ADDR} + ${RAM_SIZE})")
 zephyr_linker_symbol(SYMBOL __kernel_ram_size EXPR "(@__kernel_ram_end@ - @__bss_start@)")
 zephyr_linker_symbol(SYMBOL _image_ram_start  EXPR "(${RAM_ADDR})" SUBALIGN 32) # ToDo calculate 32 correctly
+zephyr_linker_symbol(SYMBOL _image_ram_end    EXPR "(_end)")
```

Comment:
this symbol is not being picked up, will take a closer look on what would be the correct symbol syntax for the arm linker.

Extra info, seeing this during linking:
```
Error: L6218E: Undefined symbol _end (referred from malloc.o).
```